samples/DotNet/GettingStarted/BasicSendReceiveQuickStart/BasicSendReceiveQuickStart/Program.cs

```diff
@@ -8,41 +8,41 @@ namespace BasicSendReceiveQuickStart
 {
     class Program
     {
-        static IQueueClient queueClient;
+        static IQueueClient _queueClient;
```

Comment:
Prefixing with an underscore is not a standard practice. The repo doesn't have it as a standard either.
I would suggest to remove it.

Reply:
I see, the underscore is just what I've always done for member variables. Reverted.